packages/now-client/types/types.d.ts

```diff
@@ -112,6 +112,8 @@ declare interface DeploymentOptions {
 }
 
 declare interface NowJsonOptions {
+  github?: any;
```

Comment:
Can we add an `interface` for this?

https://zeit.co/docs/v2/advanced/configuration#github.enabled